rule/rule.go

```diff
@@ -29,6 +29,7 @@ import (
 	"strings"
 
 	"github.com/elastic/go-libaudit/v2/auparse"
+	"github.com/elastic/go-libaudit/v2/internal"
```

Comment:
AFAICS the code here is only called during setup, in `auditbeat show auditd-rules`, and in testing, so it is not in a hot path and probably doesn't need this optimisation.

Reply:
removed this change from pkg `rule`.